include/ginkgo/core/solver/batch_solver_base.hpp

```diff
@@ -289,8 +289,8 @@ class EnableBatchSolver
             auto id = Identity::create(exec, system_matrix->get_size());
             preconditioner_ = std::move(id);
         }
-        const size_type workspace_size = system_matrix->get_num_batch_items() *
-                                         (sizeof(real_type) + sizeof(int));
+        const size_type workspace_size =
+            system_matrix->get_num_batch_items() * 32;
```

Comment:
still not entirely sure about where the 32 comes from, but not a blocker.

Reply:
maybe replaced it by log_data->get_workspace_size(num_batch)? It may help understanding for the future. 
At least, some comments here to indicate what this workspace for.